roles/docker/tasks/main.yml

```diff
@@ -18,6 +18,10 @@
   when: dns_mode != 'none' and resolvconf_mode == 'docker_dns'
   tags: facts
 
+- include: container_storage_setup.yml
+  when: docker_container_storage_setup and ansible_os_family == "RedHat"
```

Comment:
I would say this applies to all redhat like os:es including atomic. In my main test case it's centos 7 that needs devicemapper storage setup, but there's nothing (really) that prevents generalisation to debian family.